src/main/Thumbnail.tsx

```diff
@@ -106,7 +107,8 @@ const Thumbnail : React.FC = () => {
         discard={discardThumbnail}
       />
       <div css={bottomStyle}>
-        <VideoPlayers refs={generateRefs} widthInPercent={100}/>
+        {/* use maxHeightInPixel to make video players the same size*/}
+        <VideoPlayers refs={generateRefs} widthInPercent={100} maxHeightInPixel={376}/>
```

Comment:
This magic number depends on the width of the buttons. If a button is hidden (like in #1208), the width changes. 

![image](https://github.com/opencast/opencast-editor/assets/7419664/9b06f34b-227e-4a42-8626-5a015b5261f0)

So this should likely have a different solution. Maybe both boxes just get a fixed width. Or something else.

Reply:
Welp, that teaches me to not go for the quick and dirty solution. I'll see if I can get a proper solution working, but that'll require some time.